neuro_evolution_ctrnn/tools/env_handler.py

```diff
@@ -14,12 +14,25 @@
 class EnvHandler:
     """this class creates and modifies openAI-Environment."""
 
-    def __init__(self, config: IEpisodeRunnerCfg):
-        self.conf = config
+    def __init__(self, config: ExperimentCfg):
+        self.config = config
 
     def make_env(self, env_id: str):
-
-        env = gym.make(env_id)
+        if env_id == "ReacherMemory-v0":
+            try:
```

Comment:
ich denke diesen block kann man umschreiben als:

    assert isinstance(self.config.environment_attributes, ReacherMemoryEnvAttributesCfg) "For the environment 'ReacherMemory-v0' one must provide the ReacherMemoryEnvAttributesCfg (config.environment_attributes)"

Denn bei `assert` kann man die Fehlermeldung als 3. Parameter mitgeben (siehe z.B. [hier](https://www.w3schools.com/python/ref_keyword_assert.asp))

Reply:
:+1: